be-traveltrove/src/main/java/com/traveltrove/betraveltrove/dataaccess/tour/TourEvents.java

```diff
@@ -21,4 +23,5 @@ public class TourEvents {
     private String seqDesc;
     private String tourId; // FK tour ID
     private String events; // FK events ID
+    private String toursEventId = UUID.randomUUID().toString();
```

Comment:
Can you explain why you put the UUID generation in the entity class model instead of the POST request?